pants.toml

```diff
@@ -14,6 +14,7 @@ backend_packages = [
   "pants.backend.experimental.java.lint.google_java_format",
   "pants.backend.experimental.scala",
   "pants.backend.experimental.scala.lint.scalafmt",
+  "pants.backend.shell",
```

Comment:
FWIW, pants doesn't _need_ to track the `get-pants.sh` script. So, I'd suggest leaving this backend off, and remove the `shell_sources` out too. From my perspective, this would keep the example repo simpler/more focused.